cmd/server/main.go

```diff
@@ -1,3 +1,41 @@
 package main
 
-func main() {}
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/caarlos0/env/v11"
+	"github.com/gin-gonic/gin"
+	"github.com/xoxloviwan/go-monitor/internal/api"
+)
+
+const DA = "localhost:8080"
```

Comment:
Хорошая идея использовать константы для значений по-умолчанию, но я рекомендовал бы дать им более информативные имена:
```suggestion
const AddressDefault = "localhost:8080"
```